daemon/stores/workunit.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+from pathlib import Path
+from typing import Union
+
+from jina.helper import colored
+from .base import BaseStore
+
+
+class WorkunitStore(BaseStore):
```

Comment:
What is a `WorkunitStore`? How is it different from a `WorkspaceStore`?

Reply:
WorkunitStore is an abstract class that FlowStore, PeaStore and PodStore extends. Its main function is to hold workspace_store instance and encapsulate delete function from WorkspaceStore. When a DELETE request comes to Flow/Pea/Pod, both itself and workspace should be deleted from corresponding stores. WorkunitStore provides this behavior.